[2022-08-03 14:40:14,291] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T14:30:00+00:00 [queued]>
[2022-08-03 14:40:14,434] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T14:30:00+00:00 [queued]>
[2022-08-03 14:40:14,435] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 14:40:14,435] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 14:40:14,436] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 14:40:14,458] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-03T14:30:00+00:00
[2022-08-03 14:40:14,463] {standard_task_runner.py:51} INFO - Started process 18564 to run task
[2022-08-03 14:40:14,470] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-03T14:30:00+00:00', '--job-id', '1179', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmp4ghg0wwd']
[2022-08-03 14:40:14,475] {standard_task_runner.py:76} INFO - Job 1179: Subtask respond
[2022-08-03 14:40:14,602] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-03T14:30:00+00:00 [running]> on host b4dcda255553
[2022-08-03 14:40:14,712] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T14:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T14:30:00+00:00
[2022-08-03 14:40:14,713] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-03 14:40:14,833] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220803T143000, start_date=20220803T144014, end_date=20220803T144014
[2022-08-03 14:40:14,964] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-03 14:40:15,003] {local_task_job.py:118} INFO - Task exited with return code 0
